projects/canopy/src/lib/filter-buttons/filter-button/filter-button.component.scss

```
@@ -0,0 +1,99 @@
+@import '../../../styles/mixins';
+
+.lg-filter-button__input {
+  @include lg-visually-hidden;
+}
+
+.lg-filter-button__label {
+  display: inline-block;
+  max-height: var(--space-lg);
+  border: solid var(--border-width) var(--filter-button-bg-color-active);
+  border-radius: var(--space-md);
+  padding: var(--space-xxxs) var(--space-sm) var(--space-xxxs) var(--space-sm);
```

```suggestion
  padding: var(--space-xxxs) var(--space-sm);
```
I think this can be shortened

---

Done.